src/LedManagerTask.cpp

```diff
@@ -125,11 +171,22 @@ void LedManagerTask::begin(EvseManager &evse)
 
 void LedManagerTask::setup()
 {
-#if defined(NEO_PIXEL_PIN) && defined(NEO_PIXEL_LENGTH)
+#if defined(NEO_PIXEL_PIN) && defined(NEO_PIXEL_LENGTH) && !defined(ENABLE_WS2812FX)
   DBUGF("Initialising NeoPixels");
   strip.begin();
   //strip.setBrightness(brightness);
   setAllRGB(0, 0, 0);
+#elif defined(NEO_PIXEL_PIN) && defined(NEO_PIXEL_LENGTH) && defined(ENABLE_WS2812FX)
+  DEBUG.printf("Initialising NeoPixels WS2812FX MODE...\n");
+  ws2812fx.init();
+  ws2812fx.setBrightness(brightness);
+  ws2812fx.setSpeed(DEFAULT_FX_SPEED);
+  ws2812fx.setColor(BLACK);
+  ws2812fx.setMode(FX_MODE_STATIC);
+  //ws2812fx.setBrightness(this->brightness);
+  DEBUG.printf("Brightness: %d ", this->brightness);
+  DEBUG.printf("Brightness: %d ", brightness);
```

Comment:
Use `DBUGF` so these debug lines get compiled out in release builds. To enable the debug you need to define `ENABLE_DEBUG_LED` in `platformio.ini`
```suggestion
  DBUGF("Brightness: %d ", this->brightness);
  DBUGF("Brightness: %d ", brightness);
```